modules/ROOT/pages/endpoint-ftp-receive.adoc

```diff
@@ -48,6 +48,10 @@ Create FTP receive endpoints by configuring these settings on the *<host>* or *P
 | Port number of the FTP server.
 | x
 
+| Character encoding override
+| Enables inbound endpoints to receive non UTF-8 encoded documents for XML, CSV, and X12 document types.
```

Comment:
Please remove " for XML, CSV, and X12 document types.", as the config applies for all message formats including JSON and EDIFACT too. We do not have to mention the document types explicitly.